client/me/purchases/concierge-banner/index.jsx

```diff
@@ -107,6 +110,7 @@ class ConciergeBanner extends Component {
 			<>
 				<TrackComponentView eventName="calypso_purchases_concierge_banner_view" />
 				<Banner
+					className={ 'concierge-banner' }
```

Comment:
This would be a good candidate for a `prettier` rule. Currently, it just maintains what exists already, which can be with or without brackets. I've manually fixed it and a couple of others for now.